src/settings/OrderTemplates/OrderTemplatesEditor/OrderTemplatesEditor.js

```diff
@@ -20,6 +21,7 @@ import {
   Row,
 } from '@folio/stripes/components';
 import stripesForm from '@folio/stripes/final-form';
+import { EditCustomFieldsRecord } from '@folio/stripes/smart-components';
 import {
```

Comment:
Empty line here

Reply:
Are you sure? 
Here is the grouping like:
import ... from 'prop-types';
import ... from 'prop-types';
import ... from 'prop-types';